CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The following changes have been implemented but not released yet:
 
 ## [Unreleased]
 
+### New features
+
+- `@inrupt/solid-client/access/universal` and `@inrupt/solid-client/access/for`:
```

Comment:
```suggestion
- `@inrupt/solid-client/access/universal`:
```

The former re-exports the latter, so there's no need to expose the latter :)

Reply:
If you only used `getAccessFor`, and directly imported `@inrupt/solid-client/access/for`, wouldn't you be able to tree-shake `@inrupt/solid-client/access/universal` ? It's not like it was a massive gain, but that was my reasoning initially ^^

Reply:
Most bundlers are luckily smarter than that and can statically analyse code on the function level, i.e. if you only import a single function from a module, and that doesn't reference other functions in that module and doesn't have side effects, it will drop all other code from that module. (This only works with ES modules though.)